src/data/email_template_variables_sample.json

```diff
@@ -52,5 +53,143 @@
     "set_password_link_to_registration": "https://example-url.com",
     "need_details": false,
     "order_credit_card_type": "VISA",
-    "order_credit_card_4number": "4242"
+    "order_credit_card_4number": "4242",
+    "room_complete_name": "Room Name",
+    "reservation_start_datetime": "2023-09-01 00:00:00",
+    "reservation_end_datetime": "2023-09-01 01:00:00",
+    "reservation_created_datetime": "2023-09-01 00:00:00",
+    "reservation_amount": "500",
+    "reservation_currency": "USD",
+    "reservation_id": 123,
+    "room_capacity": 1,
+    "reservation_refunded_amount": "200.00",
+    "accepted_moderated_presentations": "accepted_moderated_presentations",
```

Comment:
this is not correct , is a collection check https://showadmin.dev.fnopen.com/app/summits/50/email-flow-events/14962